testdata/p4_14_samples_outputs/switch_20160512/switch-frontend.p4

```diff
@@ -657,166 +658,166 @@ header vlan_tag_t {
     int_ingress_tstamp_header_t             int_ingress_tstamp_header;
-    @name(".int_q_congestion_header")
+    @name(".int_q_congestion_header") 
     int_q_congestion_header_t               int_q_congestion_header;
-    @name(".int_q_occupancy_header")
+    @name(".int_q_occupancy_header") 
     int_q_occupancy_header_t                int_q_occupancy_header;
-    @name(".int_switch_id_header")
+    @name(".int_switch_id_header") 
     int_switch_id_header_t                  int_switch_id_header;
-    @name(".ipv4")
+    @name(".ipv4") 
     ipv4_t                                  ipv4;
-    @name(".ipv6")
+    @name(".ipv6") 
     ipv6_t                                  ipv6;
-    @name(".lisp")
+    @name(".lisp") 
     lisp_t                                  lisp;
-    @name(".llc_header")
+    @name(".llc_header") 
     llc_header_t                            llc_header;
-    @name(".nsh")
+    @name(".nsh") 
     nsh_t                                   nsh;
-    @name(".nsh_context")
+    @name(".nsh_context") 
     nsh_context_t                           nsh_context;
-    @name(".nvgre")
+    @name(".nvgre") 
     nvgre_t                                 nvgre;
-    @name(".outer_udp")
+    @name(".outer_udp") 
     udp_t                                   outer_udp;
-    @name(".roce")
+    @name(".roce") 
     roce_header_t                           roce;
-    @name(".roce_v2")
+    @name(".roce_v2") 
     roce_v2_header_t                        roce_v2;
-    @name(".sctp")
+    @name(".sctp") 
     sctp_t                                  sctp;
-    @name(".sflow")
+    @name(".sflow") 
     sflow_hdr_t                             sflow;
-    @name(".sflow_raw_hdr_record")
+    @name(".sflow_raw_hdr_record") 
     sflow_raw_hdr_record_t                  sflow_raw_hdr_record;
-    @name(".sflow_sample")
+    @name(".sflow_sample") 
     sflow_sample_t                          sflow_sample;
-    @name(".snap_header")
+    @name(".snap_header") 
     snap_header_t                           snap_header;
-    @name(".tcp")
+    @name(".tcp") 
     tcp_t                                   tcp;
-    @name(".trill")
+    @name(".trill") 
     trill_t                                 trill;
-    @name(".udp")
+    @name(".udp") 
     udp_t                                   udp;
-    @name(".vntag")
+    @name(".vntag") 
     vntag_t                                 vntag;
-    @name(".vxlan")
+    @name(".vxlan") 
     vxlan_t                                 vxlan;
-    @name(".vxlan_gpe")
+    @name(".vxlan_gpe") 
     vxlan_gpe_t                             vxlan_gpe;
-    @name(".vxlan_gpe_int_header")
+    @name(".vxlan_gpe_int_header") 
     vxlan_gpe_int_header_t                  vxlan_gpe_int_header;
-    @name(".int_val")
+    @name(".int_val") 
     int_value_t[24]                         int_val;
-    @name(".mpls")
+    @name(".mpls") 
     mpls_t[3]                               mpls;
-    @name(".vlan_tag_")
+    @name(".vlan_tag_") 
```

Comment:
@mbudiu-vmw Do you know what is generating these trailing white-spaces?

Reply:
Yes, the code in toP4 for annotations always adds a trailing space.
I don't know why it wasn't there before... the code has been there for 5 years.
But our diff testing tool is insensitive to whitespace, so these changes would not trigger test failures.